chromium_src/third_party/blink/renderer/core/page/chrome_client_impl.cc

```diff
@@ -0,0 +1,41 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "src/third_party/blink/renderer/core/page/chrome_client_impl.cc"
+
+#include "brave/third_party/blink/renderer/farbling/brave_session_cache.h"
+#include "src/third_party/blink/renderer/core/execution_context/execution_context.h"
```

Comment:
why are you using `src/` for all of these? That should only be used to get the original copy without chromium_src overrides.

Reply:
and we don't even have chromium_src overrides for some of these files

Reply:
Fixed

Reply:
fixed